.github/workflows/verify.yml

```diff
@@ -51,6 +51,7 @@ jobs:
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
```

Comment:
Is there any more context on why this change was needed? 👀 

From a quick Google it looks like folk are adding this when they're wanting their github action to work with `pull_request_target` - which isn't the case here:

https://github.com/packetfu/packetfu/blob/bc7362f7c3d8e40e2c8bdeaab76b7c17506306c1/.github/workflows/verify.yml#L19-L25